src/main/resources/com/google/api/codegen/php/main.snip

```diff
@@ -486,8 +486,17 @@
             {@""} * @@param array $optionalArgs {
              *     Optional.
             @join param : optionalFields
-                @join commentLine : context.php.getDocLinesWithPrefixes(context.getDescription(param), optionalParamDocLinePrefix(param), "          ")
-                    {@commentLine}
+                @if isPageSizeField(param, methodConfig)
+                    {@""} *     @@type integer ${@methodConfig.getPageStreaming.getPageSizeField.getSimpleName}
```

Comment:
We do this only if the GAPIC yaml annotates that particular field as belonging to page streaming. This is necessary because VKit page streaming behavior is not the same as gRPC page streaming; in many cases the proto comments on this field simply do not apply to VKit code, for example, if you are doing per-resource page streaming.

I think doing this is OK because this is a field with common semantics across VKit libraries in the specific context of page streaming.